openlibrary/i18n/id/messages.po

```diff
@@ -0,0 +1,5477 @@
+
+#: diff.html:26
+msgid "Added"
+msgstr "Ditambah"
+
+#: diff.html:27
+msgid "Modified"
+msgstr "Diubah"
+
+#: diff.html:28
+msgid "Removed"
+msgstr "Dihapus"
+
+#: diff.html:29
+msgid "Not changed"
+msgstr "Tidak berubah"
+
+#: diff.html:39
+#, python-format
+msgid "Revision %d"
+msgstr "Revisi nomor %d"
+
+#: books/check.html:32 books/edit/edition.html:76 books/show.html:16
+#: covers/book_cover.html:41 covers/book_cover_single_edition.html:36
+#: covers/book_cover_work.html:32 diff.html:42 lists/preview.html:20
+msgid "by"
+msgstr "oleh"
+
+#: diff.html:44
+msgid "by Anonymous"
+msgstr "oleh Anonymous"
+
+#: diff.html:110
+msgid "Differences"
+msgstr "Perbedaan"
+
+#: diff.html:168
+msgid "Both the revisions are identical."
+msgstr "Kedua revisi tidak memiliki perbedaan."
+
+#: edit_yaml.html:14 lib/edit_head.html:17
+msgid "You're in edit mode."
+msgstr "Anda sedang dalam mode edit."
+
+#: edit_yaml.html:15 lib/edit_head.html:18
+msgid "Cancel, and go back."
+msgstr "Batal, lalu kembali."
+
+#: edit_yaml.html:22
+msgid "YAML Representation:"
+msgstr "Representasi YAML:"
+
+#: BookPreview.html:11 books/edit/edition.html:669 editpage.html:15
+msgid "Preview"
+msgstr "Pratinjau"
+
+#: history.html:23
+msgid "History of edits to"
+msgstr "Riwayat penyuntingan untuk"
+
+#: history.html:31
+msgid "Revision"
+msgstr "Revisi"
+
+#: RecentChanges.html:17 RecentChangesAdmin.html:20 RecentChangesUsers.html:20
+#: admin/ip/view.html:44 admin/people/edits.html:41 history.html:32
+#: recentchanges/render.html:30 recentchanges/updated_records.html:28
+msgid "When"
+msgstr "Kapan"
+
+#: RecentChanges.html:19 admin/ip/view.html:46 admin/loans_table.html:46
+#: admin/people/edits.html:43 admin/waitinglists.html:28 history.html:33
+#: recentchanges/render.html:32
+msgid "Who"
+msgstr "Siapa"
+
+#: RecentChanges.html:20 RecentChangesUsers.html:22 admin/ip/view.html:47
+#: admin/people/edits.html:44 history.html:34 recentchanges/render.html:33
+#: recentchanges/updated_records.html:30
+msgid "Comment"
+msgstr "Komentar"
+
+#: history.html:35
+msgid "Diff"
+msgstr "Perbedaan"
+
+#: history.html:42 history.html:73
+msgid "Compare"
+msgstr "Bandingkan"
+
+#: history.html:42 history.html:73
+msgid "See Diff"
+msgstr "Lihat perbedaan"
+
+#: history.html:48 lib/history.html:76
+#, python-format
+msgid "View revision %s"
+msgstr "Lihat revisi nomor %s"
+
+#: history.html:85
+msgid "Back"
+msgstr "Kembali"
+
+#: Pager.html:34 history.html:87 lib/pagination.html:37 showmarc.html:54
+msgid "Next"
+msgstr "Berikutnya"
+
+#: internalerror.html:14
+msgid "Sorry. There seems to be a problem with what you were just looking at."
+msgstr "Maaf. Ada masalah dengan halaman yang sedang Anda lihat."
+
+#: internalerror.html:15
+#, python-format
+msgid ""
+"We've noted the error %s and will look into it as soon as possible. Head for <a "
+"href=\"/\">home</a>?"
+msgstr ""
+"Kami telah menemukan error %s dan akan memeriksanya segera. Kembali ke <a href=\"/"
+"\">home</a>?"
```

Comment:
Is the English word "home" used for web homepages in Indonesian, or is this a missed translation?

Reply:
@IrennaLumbuun, are you able to comment on this? I'd dearly like to get this PR into a place where we can merge it, even if it is incomplete and not ready to enable on the main site, if only so it's easier to review when changes are added in the future.

Reply:
Looks like I missed this when making the change. Let me get back on it in a couple days! Thx for following up :)

Reply:
Thanks for the quick response!

Reply:
No worries! I also marked this PR as ready so it can be merged